zammad/templates/deployment-railsserver.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "zammad.fullname" . }}-railsserver
+  labels:
+    {{- include "zammad.labels" . | nindent 4 }}
+    app.kubernetes.io/component: zammad-railsserver
+spec:
+  replicas: {{ .Values.zammadConfig.railsserver.replicas }}
+  selector:
+    matchLabels:
+      {{- include "zammad.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "zammad.labels" . | nindent 8 }}
+        app.kubernetes.io/component: zammad-railsserver
+    spec:
+      {{- include "zammad.podSpec" . | nindent 6 }}
+      {{- if .Values.zammadConfig.initContainers.volumePermissions.enabled }}
+      initContainers:
+        - name: zammad-volume-permissions
+          image: "{{ .Values.zammadConfig.initContainers.volumePermissions.image.repository }}:{{ .Values.zammadConfig.initContainers.volumePermissions.image.tag }}"
+          imagePullPolicy: {{ .Values.zammadConfig.initContainers.volumePermissions.image.pullPolicy }}
+          command:
+            - /bin/sh
+            - -cx
+            - chmod 770 /opt/zammad/tmp
+          {{- with .Values.zammadConfig.initContainers.volumePermissions.resources }}
+          resources:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          {{- with .Values.zammadConfig.initContainers.volumePermissions.securityContext }}
+          securityContext:
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          volumeMounts:
+            {{- include "zammad.volumeMounts" . | nindent 12 }}
+      {{- end }}
```

Comment:
I am not entirely sure about this part, but it seems to be required to fix up the volume permissions for the container (so that `tmp/` is not world-writable).